fail2ban/server/filter.py

```diff
@@ -369,16 +373,20 @@ def inIgnoreIPList(self, ip):
 				# Check if IP in DNS
 				ips = DNSUtils.dnsToIp(i)
 				if ip in ips:
+					self.logIgnoreIp(ip, log_ignore, ignore_source="ignoreip by dns")
 					return True
 				else:
 					continue
 			if a == b:
+				self.logIgnoreIp(ip, log_ignore, ignore_source="ignoreip by addr")
 				return True
 
 		if self.__ignoreCommand:
 			command = CommandAction.replaceTag(self.__ignoreCommand, { 'ip': ip } )
 			logSys.debug('ignore command: ' + command)
-			return CommandAction.executeCmd(command)
+			ret_ignore = CommandAction.executeCmd(command)
+			self.logIgnoreIp(ip, log_ignore and ret_ignore, ignore_source="ignorecommand")
```

Comment:
I was thinking there were two reasons, one of which had two ways to determine (dns/ip)...but I like the simpler approach you suggested - https://github.com/leeclemens/fail2ban/commit/f4341999cd5e5695e600e051ac7fdf0edffbb93c